docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_augmented_billable_hcc_en.md

```diff
@@ -0,0 +1,179 @@
+| subsequent type two diabetes mellitus | PROBLEM  | E11          | E11:::E11.62:::E11.5:::E11.69:::E11.59:::E09:::E11.6:::E11.8...                        | type 2 diabetes mellitus [type 2 diabetes mellitus]:::type 2 diabetes mellitus with skin complications [type 2 diabetes mellitus with skin complications]:::type 2 diabetes mellitus with circulatory complications [type 2 diabetes mellitus with circulatory complications]:::type 2 diabetes mellitus with other specified complication [type 2 diabetes mellitus with other specified complication]...                                                                                                       | '0||0||0', '0||0||0', '0||0||0', '1||1||18', '1||1||18', '0||0||0', '0||0||0', '1||1||18', '0||0||0', '1||1||18', '0||0||0', '1||1||18', '1||1||18', '0||0||0', '0||0||0', '1||1||18', '0||0||0', '0||0||0', '1||1||18', '0||0||0', '1||0||0', '0||0||0', '1||1||18'                       |
+| T2DM                                  | PROBLEM  | Q06.0        | Q06.0:::N94.0:::B48.3:::L02.42:::R14.2:::L02.43:::R29.702:::H53.55...                  | amyelia [amyelia]:::mittelschmerz [mittelschmerz]:::geotrichosis [geotrichosis]:::furuncle of limb [furuncle of limb]:::eructation [eructation]:::carbuncle of limb [carbuncle of limb]:::nihss score 2 [nihss score 2]:::tritanomaly [tritanomaly]:::beriberi [beriberi]:::megaloureter [megaloureter]...                                                                                                                                                                                                       | '1||1||72', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0', '0||0||0', '0||0||0', '1||1||86', '1||0||0', '1||0||0', '1||0||0', '1||0||0'        |
+| HTG-induced pancreatitis              | PROBLEM  | K85.3        | K85.3:::K85:::K86.0:::K85.2:::K85.1:::K85.0:::K85.32:::K85.8:::B25.2...                | drug induced acute pancreatitis [drug induced acute pancreatitis]:::acute pancreatitis [acute pancreatitis]:::alcohol-induced chronic pancreatitis [alcohol-induced chronic pancreatitis]:::alcohol induced acute pancreatitis [alcohol induced acute pancreatitis]:::biliary acute pancreatitis [biliary acute pancreatitis]:::idiopathic acute pancreatitis [idiopathic acute pancreatitis]...                                                                                                                 | '0||0||0', '0||0||0', '1||1||34', '0||0||0', '0||0||0', '0||0||0', '1||0||0', '0||0||0', '1||1||6', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '1||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0', '1||0||0', '1||1||34', '0||0||0', '1||0||0', '1||0||0', '0||0||0', '0||0||0'        |
```

Comment:
remove/change this chunk with `drug-induced pancreatitis`

Reply:
Done!